api4/cloud.go

```diff
@@ -38,6 +38,9 @@ func (api *API) InitCloud() {
 
 	// POST /api/v4/cloud/webhook
 	api.BaseRoutes.Cloud.Handle("/webhook", api.CloudApiKeyRequired(handleCWSWebhook)).Methods("POST")
+
+	// GET /api/v4/cloud/free_tier/stats
+	api.BaseRoutes.Cloud.Handle("/free_tier/stats", api.ApiSessionRequired(getFreeTierStats)).Methods("GET")
```

Comment:
Given that these stats are related to their subscription, maybe we can follow the existing convention and have the endpoint be `/api/v4/cloud/subscription/stats`?
```suggestion
	api.BaseRoutes.Cloud.Handle("/subscription/stats", api.ApiSessionRequired(getSubscriptionStats)).Methods("GET")
```